plugins/modules/gitlab_group.py

```diff
@@ -279,12 +286,15 @@ def update_group(self, group, arguments):
 
         return (changed, group)
 
-    def delete_group(self):
+    '''
+    @param force To delete even if projects inside
+    '''
+    def delete_group(self, force=False):
         group = self.group_object
 
-        if len(group.projects.list(all=False)) >= 1:
+        if not force and len(group.projects.list(all=False)) >= 1:
             self._module.fail_json(
-                msg="There are still projects in this group. These needs to be moved or deleted before this group can be removed.")
+                msg="There are still projects in this group. These needs to be moved or deleted before this group can be removed. Use 'force_delete' to 'true' to force deletion existing projects.")
```

Comment:
```suggestion
                msg="There are still projects in this group. These needs to be moved or deleted before this group can be removed. Use 'force_delete' to 'true' to force deletion of existing projects.")
```
This needs to be split up into two (or more) lines as well since it's too long (> 160 chars).